scripts/local_k8s/config.py

```
@@ -1,53 +1,97 @@
     chart_path: Path
     local_values_path: Path
     dockerfile_path: Path
 
+    @property
+    def kube_context(self) -> str:
+        """Return the kube context name created by the selected provider."""
+
+        return f"{self.k8s_provider}-{self.cluster_name}"
+
     @classmethod
     def from_env(cls) -> "PreviewConfig":
```

Ruff (0.15.18)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)